contents/teams/error-tracking/objectives.mdx

```diff
@@ -0,0 +1,20 @@
+### 🎯 Objective: Human readable stack traces for Python
```

Comment:
and JS? 🙈

Reply:
Whoopsies... wrong way around. Python stack traces are already readable, JS _can be_ minified